src/commands/startStreamingLogs.ts

```diff
@@ -6,19 +6,16 @@
 import * as vscode from 'vscode';
 import * as appservice from 'vscode-azureappservice';
 import { IActionContext } from 'vscode-azureextensionui';
-import { LogStreamTreeItem } from '../explorer/LogStreamTreeItem';
 import { SiteTreeItem } from "../explorer/SiteTreeItem";
 import { WebAppTreeItem } from '../explorer/WebAppTreeItem';
 import { ext } from '../extensionVariables';
 import { enableFileLogging } from './enableFileLogging';
 
-export async function startStreamingLogs(context: IActionContext, node?: SiteTreeItem | LogStreamTreeItem): Promise<void> {
+export async function startStreamingLogs(context: IActionContext, node?: SiteTreeItem): Promise<void> {
     let siteTreeItem: SiteTreeItem;
 
     if (!node) {
         siteTreeItem = <WebAppTreeItem>await ext.tree.showTreeItemPicker(WebAppTreeItem.contextValue, context);
```

Comment:
Was it okay to remove the check for getting a SiteTreeItem?  What if they enter through the LogStreamTreeItem?

Reply:
I modified commandArgs so that it would pass the parent instead of the LogStreamTreeItem itself:
https://github.com/microsoft/vscode-azuretools/pull/613/files#diff-81088792bebe528022540e6bb9a4512aR34